tensorboard/webapp/plugins/plugins_component.ng.html

```diff
@@ -21,38 +21,66 @@
   <!-- render Angular built-in plugins with a dynamic component resolver. -->
   <ng-container #ngPluginContainer></ng-container>
 </div>
-<div *ngIf="noEnabledPlugin" class="no-plugin">
+
+<div
+  *ngIf="pluginLoadState !== PluginLoadState.LOADED && pluginLoadState !== PluginLoadState.LOADING"
```

Comment:
Since the enum of states differ from the standard not_loaded/loaded/loading/failed, can we instead write the condition in terms of the positive matches?

e.g. ngIf pluginLoadState === UNKNOWN_PLUGIN_ID || ...
?